### DISCUSSION TITLE
Validation errors in DatePicker

### DISCUSSION THREAD

**Discussion by benain** [Megabit/Blazorise#5474] [Q&A] [answered]
Hi all,

I've been struggling for a while trying to implement a validation into a set of datepickers that show the period of validity of an item.
I place the Feedback inside the DatePickers, but they are never visible even when I return error. Is this a limitation of the DatePicker due to being a wrapper?
![image](https://github.com/Megabit/Blazorise/assets/12417750/121de532-2520-4959-8189-779c5b496fec)

Thank you in advance for your replies!

**Comment by stsrki** [comment, marked as answer]
What version are you using? This should have been fixed in the latest 1.5.2.

**Reply by benain** [reply]
Oh God... I still had the 1.5.1!! 🤦‍♂️
Many many thanks for your answer and time! You really made magic with Blazorise!
